src/sentry/notifications/notificationcontroller.py

```diff
@@ -275,6 +276,19 @@ def _get_layered_setting_providers(
             )
         )
 
+        org = (
+            serialize_organization_mapping(
+                OrganizationMapping.objects.filter(organization_id=self.organization_id).first()
+            )
+            if self.organization_id
+            else None
+        )
+        has_team_workflow = (
+            True
+            if org and features.has("organizations:team-workflow-notifications", org)
+            else False
+        )
```

Comment:
```suggestion
        has_team_workflow = org and features.has("organizations:team-workflow-notifications", org)
```
this should be a bool on its own iirc